### DISCUSSION TITLE
The ideas about pomodoro

### DISCUSSION THREAD

**Discussion by btk-42** [almarklein/timetagger#231] [Ideas]
Hi, 

You could add option to set specific time, e.q. 20 min work 10 min break. 
You could also add a tittle for current period of work. I mean a field where user can write the topic of currently work/study. And it will    be automatically appear in main timeline.

**Comment by almarklein** [comment]
Thanks for the suggestions! We collect ideas for improving the Pomodoro at #115. I will add your suggestions there.
